go.sum

```diff
@@ -0,0 +1,12 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```

Comment:
https://stackoverflow.com/questions/53837919/should-go-sum-file-be-checked-in-to-the-git-repository